com.insynchq.Insync.yml

```diff
@@ -0,0 +1,49 @@
+app-id: com.insynchq.Insync
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: run.sh
+separate-locales: false
+rename-desktop-file: insync.desktop
+rename-icon: insync
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=network
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+    # Tray icon
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=org.kde.StatusNotifierItem
+  - --own-name=org.kde.*
+  # Needed for /dev/disk/by-id which insync accesses
+  - --device=all
+  - --filesystem=~/Insync
+  - --persist=.config
+  - --filesystem=xdg-config/autostart:create
+build-options:
+  strip: true
+modules:
+  - name: insync
+    buildsystem: simple
+    build-commands:
+      - ar x insync.deb
+      - tar xf data.tar.gz
+      - mv usr $FLATPAK_DEST
+      - cp -r $FLATPAK_DEST/usr/share $FLATPAK_DEST
+      - desktop-file-edit --set-key=Exec --set-value="run.sh start --no-daemon" $FLATPAK_DEST/share/applications/insync.desktop
```

Comment:
1. if you need to pass static parameters like that put then in the wrapper script
2. I'm pretty it doesn't even work at the moment given the wrapper script. Did you really test it? (`flatpak run` isn't the way)